pkg/interface/config/webpack.dev.js

```diff
@@ -45,21 +45,21 @@ class UrbitShipPlugin {
 let devServer = {
   contentBase: path.join(__dirname, '../dist'),
   hot: true,
-  port: 9000,
+  port: 9001,
```

Comment:
@liam-fitzgerald @matildepark this is not permanent — I accidentally left it in, but I think the ServiceWorker cached something that left 9000 in a broken state on my machine